routes/web.php

```diff
@@ -10,6 +10,7 @@
 Route::controller(MovieController::class)->group(function () {
     Route::get('/', 'index')->name('home');
     Route::get('/m/{id}/{title}', 'show')->name('movie');
+    Route::delete('/m{id}/{title}', 'destroy')->middleware(['auth'])->name('movie.destroy');
```

Comment:
this should be amongst the admin controllers! and then there's no need to add the middleware since it will be under the `admin` middleware

```suggestion
    Route::delete('/m/{id}/{title}', 'destroy')->name('movie.destroy');
```

Reply:
Should it be an separate delete-movie.blade.php file for this?